src/tools/annotation/LengthTool.test.js

```diff
@@ -182,15 +183,15 @@ describe('LengthTool.js', () => {
       };
 
       instantiatedTool.updateCachedStats(image, element, data);
-      expect(data.length.toFixed(2)).toEqual('50.86');
+      expect(data.length).toEqual(new Decimal(50.9));
 
       data.handles.start.x = 138.74809160305347;
       data.handles.start.y = 71.32824427480917;
       data.handles.end.x = 79.14503816793899;
       data.handles.end.y = 121.16030534351145;
 
       instantiatedTool.updateCachedStats(image, element, data);
-      expect(data.length.toFixed(2)).toEqual('69.80');
+      expect(data.length).toEqual(new Decimal(69.8));
     });
   });
```

Comment:
It seems like the new logic did not get tests

Reply:
hmm could you please explain further for better understanding? :)

Reply:
Ups, sorry I did not see this comment before 😓